way_to_home/static/src/components/loginBtn/loginBtn.css

```diff
@@ -1,6 +1,6 @@
 .LoginBtnDiv{
-    position: fixed;
-    right: 150px;
-    top: 50px;
+    position: absolute;
```

Comment:
fixed instead of absolute

Reply:
'login button' belongs to the 'header' components, so position absolute will follow to parent